api/src/main/java/org/apache/iceberg/encryption/EncryptedKey.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.encryption;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+public interface EncryptedKey {
```

Comment:
Unless I'm missing something, I wonder whether this should be called `EncryptionKey` since this is the key you encrypt table metadata with? Because `EncryptedKey` sounds like the key that was actually encrypted

Reply:
This key is actually encrypted :)

Reply:
ok if that's the case then I probably read the spec slightly wrong. Thanks for clarifying

Reply:
The spec uses `encryption-keys` but each key is an encrypted key.